com.transmissionbt.Transmission.appdata.xml

```diff
@@ -157,7 +157,7 @@
         <p>Fix CVE-2018-5702 security vulnerability (see https://bugs.chromium.org/p/project-zero/issues/detail?id=1447 and https://github.com/transmission/transmission/pull/468 for more details).</p>
       </description>
     </release>
-    <release date="2017-12-13" version="2.92-3">
+    <release date="2018-01-24" version="2.93">
```

Comment:
Please add a new entry with the new news.

Reply:
I meant to add a new entry instead of changing the date & version on the old entry.